waspc/data/Generator/templates/server/tsconfig.json

```diff
@@ -1,5 +1,6 @@
+{{={= =}=}}
 {
-  "extends": "@tsconfig/node18/tsconfig.json",
+  "extends": "@tsconfig/node{= majorNodeVersion =}/tsconfig.json",
```

Comment:
Is this looking ok @sodic @infomiho ?

Reply:
It looks okay. It takes configs from here: https://github.com/tsconfig/bases/tree/main/bases

For reference, 18 vs 20 don't have any changes:
![Screenshot 2023-10-25 at 13 16 02](https://github.com/wasp-lang/wasp/assets/2223680/7f8acdc8-af29-45a9-ad67-98303f5e5658)